modal/secret.py

```diff
@@ -206,13 +210,7 @@ async def lookup(
         environment_name: Optional[str] = None,
         required_keys: List[str] = [],
     ) -> "_Secret":
-        """Lookup a secret with a given name
-
-        ```python
-        s = modal.Secret.lookup("my-secret")
-        print(s.object_id)
-        ```
-        """
+        """mdmd:hidden"""
```

Comment:
I don't think users have any reason to `.lookup` a Secret? Could be wrong though...